src/components/fields/Percentage/SideDrawerField.tsx

```diff
@@ -34,11 +35,14 @@ export default function Percentage({
                 width: 20,
                 height: 20,
                 borderRadius: 0.5,
-                boxShadow: (theme) =>
-                  `0 0 0 1px ${theme.palette.divider} inest`,
+                boxShadow: `0 0 0 1px ${theme.palette.divider} inest`,
                 backgroundColor:
                   typeof value === "number"
-                    ? resultColorsScale(value).toHex() + "!important"
+                    ? resultColorsScale(
```

Comment:
Just note that in the MUI `sx` prop, you can pass in a function with `theme` as the first argument to access the theme object for any CSS property, so you didn’t need to change the `boxShadow` definition here or use `useTheme`

Reply:
I did this change because I used the theme variable for the backgroundColor as well.

Reply:
Yes, I meant you could do the same thing for `backgroundColor` as well: `backgroundColor: (theme) => ...`